src/styles.scss

```diff
@@ -2,3 +2,37 @@
 
 html, body { height: 100%; }
 body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
+
+
+:root{
+    --primary-color:  rgba(132, 52, 225, 1);
+    --secondary-color:rgba(132, 52, 225, 1);
+    --white-color: white;
+    --accent-color: #000;
+    --gray-color: gray;
+  --padding: 10px;
+  --gap-small: 10px;
+  --gap-large: 20px;
+}
+
+
+.primary-color{
+    color: var(--primary-color);
+}
+
+.white-color{
+    color: var(--white-color);
+}
+
+
+@mixin flex-between{
+    display: flex;
+align-items: center;
+justify-content: space-between;
+}
+
+@mixin flex-center{
+    display: flex;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
would be nice to use a formatting tool to format your codes for consistency and improved readability.